pressbooks/pressbooks-book
# Collapsible sections improvements

Fix for #794 

Previously, media attributions, footnotes, and about the authors content were hidden if the last section was collapsed.
This PR improves checks for collapsible sections to prevent this from happening.

#### How to test

1. Create a chapter with at least one heading 1 element.
2. Add one or more footnotes to the chapter.
3. Go to theme options -> web options -> collapse sections and check the option.
4. View the web book and click a footnote. Make sure it works as expected when sections are collapsed.
5. Remove the footnotes. 
6. Add a media with attributions.
7. Go to theme options -> global options -> media attributions and check it.
8. View the web book and make sure Media Attributions content appears when sections are collapsed.
9. Remove the media with attributions.
10. Add an author to the chapter.
11. Go to theme options -> global options -> about the author and check it.
12. View the web book and make sure About The Author content appears when sections are collapsed.

* Test them separately and together to make sure they work as expected.
